src/Lecture1.hs

```diff
@@ -119,4 +139,13 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> [Char]
+lowerAndGreater n list = go 0 0 list
+    where
+        go :: Int -> Int -> [Int] -> [Char]
+        go greater lower l
+            | null l      = show n ++ " is greater than " ++ show greater ++ " elements and lower than " ++ show lower ++ " elements"
+            | n == head l = go greater lower (tail l)
+            | n > head l  = go (greater + 1) lower (tail l)
+            | n < head l  = go greater (lower + 1) (tail l)
```

Comment:
I believe, GHC produces a warning here. It's because it doesn't know that all these checks are enough to cover all cases. Usually, the last guard is supposed to be under `otherwise`

I also see that GitHub Actions CI is not enabled for your repository. You can enable it here:

* https://github.com/thatverygeorge/exercises/actions